<!-- -->
<launch>
    <node
            type="tracked_objects_adapter.py"
            pkg="adeye"
            name="tracked_objects_adapter"
            respawn="false"
            output="screen"
    />
</launch>
